cw-deb.mk

```diff
@@ -35,68 +35,62 @@
-#                       (e.g., sprout)
-# DEB_MAJOR_VERSION   - major version number for deb package (e.g., 1.0)
-# DEB_NAMES           - space-separated base names of deb packages
-#                       (e.g., sprout sprout-dbg)
-# DEB_ARCH (optional) - override deb architecture (defaults to this
-#                       host's native architecture). E.g., set to all
-#                       for arch-independent debs.
+# PKG_COMPONENT or DEB_COMPONENT (DEB_COMPONENT takes precedence)
+#                      - name of overall component for manifest
+#                        (e.g., sprout)
+# PKG_MAJOR_VERSION or DEB_MAJOR_VERSION (DEB_MAJOR_VERSION takes precedence)
+#                      - major version number for package (e.g., 1.0)
+# PKG_NAMES or DEB_NAMES (DEB_NAMES takes precedence)
+#                      - space-separated base names of packages
+#                        (e.g., sprout sprout-dbg)
 
 # Caller may also set the following:
-# REPO_DIR             - path to repository to move Debian packages to (default
-#                        is unset, meaning don't move packages)
-# REPO_SERVER          - username and server to scp Debian pacakges to (default
-#                        is unset, meaning move packages locally)
-# CW_SIGNED            - whether to sign the generated Debian repository (default
+# PKG_MINOR_VERSION or DEB_MINOR_VERSION (DEB_MINOR_VERSION takes precedence)
+#                      - minor version number for package (default is current timestamp)
+# REPO_DIR             - path to repository to move packages to (default is unset,
+#                        meaning don't move packages)
+# REPO_SERVER          - username and server to scp pacakges to (default is unset,
+#                        meaning move packages locally)
+# CW_SIGNED            - whether to sign the generated package repository (default
 #                        is unset, meaning don't sign; set to Y to sign).
 #                        IMPORTANT: this signs the repo itself, which means that
 #                        *all* packages in it are marked authentic, not just the
-#                        ones we built just now. See http://wiki.debian.org/SecureApt
-#                        for details.
-# HARDENED_REPO_DIR    - path to the hardened repository to move Debian packages to
+#                        ones we built just now.
+# HARDENED_REPO_DIR    - path to the hardened repository to move packages to
 #                        (default is unset, meaning don't move packages)
 # HARDENED_REPO_SERVER - username and server for the hardened repository to scp
-#                        Debian pacakges to (default is unset, meaning move
+#                        packages to (default is unset, meaning move
 #                        packages locally)
 
-DEB_VERSION ?= ${DEB_MAJOR_VERSION}-$(shell date +%y%m%d.%H%M%S)
-DEB_VERSION := $(DEB_VERSION)
-ifeq ($(origin DEB_ARCH), undefined)
-DEB_ARCH := $(shell dpkg --print-architecture)
-endif
```

Comment:
Was `DEB_ARCH` only used for moving packages? (fine if that's the case, I just want to understand why it's safe to remove it).